### ISSUE TITLE
Tensorflow 1.13: please do not deprecate important functions!!!

### ISSUE THREAD

**Issue by chwang85** [tensorflow/tensorflow#26151]
According to Tensorflow 1.13, many important functions will be deprecated, such as:
tf.layers.dense()
tf.layers.dropout()
tf.layers.flatten()
tf.layers.batch_normalization()
...
Since these functions are very widely used, and also very helpful for building complicted models, would you please keep them in the future versions?

**Comment by kakakaku00** [comment]
弃用一定有他的道理

**Comment by chwang85** [comment]
Deprecating these functions is deprecating users.
Do you really understand "道理"?

**Comment by chwang85** [comment]
Please pay more attention to those unsolved issues (i.e. #6391, #22543, #26625) rather than deprecating widely used functions!

**Comment by mihaimaruseac** [comment]
Obsoleted by #26844

**Comment by chwang85** [comment]
By the way, before deprecating old components, would you please complete the current functionality? See #27042. It seems that TF is very good at deprecating widely used functions, but poor at improving its weak points
@kakakaku00 @mihaimaruseac @ymodak @fchollet

**Comment by chwang85** [comment]
When will you deprecate keras?
When will you deprecate Tensorflow?
When will you deprecate Google?
